scss/_material-checkbox.scss

```diff
@@ -23,92 +23,82 @@ $md-checkmark-color: $white;
   }
 
   label {
-    margin-bottom: 0;
     padding-left: $md-checkbox-size + 5px;
+    margin-bottom: 0;
   }
 
-  i.md-checkbox-control {
+  .md-checkbox-control {
```

Comment:
Why drop the `i` ?

Reply:
Because of parent `.md-checkbox` it's not needed and shouldn't target directly a tag, but a class as must as possible.

Reply:
Because Bootstrap doesn't accept selecting a tag element with a class at the same time, they use this rule : https://stylelint.io/user-guide/rules/selector-no-qualifying-type

Reply:
Ok thanks!